pkg/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/containerd/errdefs/pkg
+
+go 1.23.1
```

Comment:
Can this be max go1.22?
The other go.mod is 1.20.

Reply:
Updated